.github/workflows/phase_2_harbor.yml

```diff
@@ -0,0 +1,221 @@
+---
+name: Phase 2 - Harbor
+on:
+  push:
+    paths:
+      - .github/workflows/phase_2_harbor.yml
+
+env:
+  HARBOR_TAG: 2.11.2
+  PARLAY_VERSION: 0.6.0
+  SBOMASM_VERSION: 0.1.5
+  SBOMQS_VERSION: 0.1.9
+  TRIVY_VERSION: 0.54.1
+
+jobs:
+  Generate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Trivy
+        run: |
+          curl -L -o /tmp/trivy.tgz \
+            "https://github.com/aquasecurity/trivy/releases/download/v${TRIVY_VERSION}/trivy_${TRIVY_VERSION}_Linux-64bit.tar.gz"
+          tar xvf /tmp/trivy.tgz -C /tmp
+          chmod +x /tmp/trivy
+
+      - name: Checkout Harbor
```

Comment:
I would suggest using a [release download](https://github.com/goharbor/harbor/releases) here instead of cloning the repo. You can pin the release using the environment variables (see the example of Trivy above).